app/models/partner.rb

```diff
@@ -0,0 +1,4 @@
+class Partner < ApplicationRecord
+  belongs_to :assessment, optional: true
+  validates :date_of_birth, comparison: { less_than_or_equal_to: Date.current }
```

Comment:
@starswan I was unsure about this: It's not an error per se  - this isn't broken, and it follows a pattern common to other CFE models. It's just that in the other PR it turned out that the custom validator offered an error message that was more appropriate to show to a client.

So on balance I felt like the refactor to use the custom validator belonged in the other PR, because it was fulfilling that ticket's requirements. Whereas this code fulfils this ticket's.

Reply:
Nah, I'm splitting hairs, will just change this.